include/swift/Driver/Action.h

```diff
@@ -273,10 +273,16 @@ class ModuleWrapJobAction : public JobAction {
 
 class AutolinkExtractJobAction : public JobAction {
```

Comment:
I dont understand why this needs to be changed for supporting LTO.  The autolinking emulation for ELF should be unimpacted by this.

Reply:
Can you please add the additional context on _why_ the autolink extract job needs to care about object files vs bitcode files?  The auto linking directives in the bitcode files should never matter.  When the LTO operation occurs the embedded options will be serialized to the object files and should be extract from that.

Reply:
I was talking about "input file selection **before** invoking `swift-autolink-extract` command". Sorry for misleading.

But eventually, I changed my mind that the driver should pass both object files and bitcode files.